config_defaults/subtests/docker_cli/run.ini

```diff
@@ -54,6 +54,20 @@ last_name_sticks = yes
 names_count = 1000
 exit_status = 0
 
+[docker_cli/run/run_tmpfs]
+cmd = 'sleep 5s'
```

Comment:
Ed's right, using sleep sucks (I forgot that was in there).  Better to block on a much more deterministic condition (like a docker command exiting).  There are way too many tests that do the sleep thing, but most were written in haste and desperation - terrible examples to follow.  Sorry about that.